governance/default.go

```diff
@@ -521,18 +521,35 @@ func (g *Governance) ClearVotes(num uint64) {
 // parseVoteValue parse vote.Value from []uint8 to appropriate type
 func (g *Governance) ParseVoteValue(gVote *GovernanceVote) (*GovernanceVote, error) {
 	var val interface{}
-	k := GovernanceKeyMap[gVote.Key]
+	k, ok := GovernanceKeyMap[gVote.Key]
+	if ok == false {
+		logger.Warn("Unknown key was given", "key", k)
+		return nil, ErrUnknownKey
+	}
+
+	if (k == params.AddValidator || k == params.RemoveValidator) && (reflect.TypeOf(gVote.Value) != reflect.TypeOf([]interface{}{}) && reflect.TypeOf(gVote.Value) != reflect.TypeOf([]uint8{})) {
```

Comment:
Wouldn't it be better to make a switch statement for this and the next conditional statements?

Reply:
Thanks for the nice advice! reflected at https://github.com/klaytn/klaytn/pull/1110/commits/680be2ebb4e2fc84f7ecab4319657906761f9377